http-clients/apache5-client/src/test/java/software/amazon/awssdk/http/apache5/Apache5ClientTlsAuthTest.java

```diff
@@ -252,4 +258,76 @@ private HttpExecuteResponse makeRequestWithHttpClient(SdkHttpClient httpClient)
         return httpClient.prepareRequest(request).call();
     }
 
+    @Test
+    public void tls_strategy_configuration() throws Exception {
```

Comment:
minor: can we fix the test names so they match our normal conventions? i.e. `methodToTest_when_expectedBehavior`